shell/platform/windows/text_input_plugin.cc

```diff
@@ -22,6 +22,7 @@ static constexpr char kHideMethod[] = "TextInput.hide";
 static constexpr char kSetMarkedTextRect[] = "TextInput.setMarkedTextRect";
```

Comment:
Is this now unused? If so, it would need to be removed, but please see my comment below since at least for Japanese input, the candidate menu shouldn't move along with the cursor.